packages/core/src/notifications/panelNode.ts

```diff
@@ -78,10 +78,19 @@ export class NotificationsNode implements TreeNode {
 
     public getChildren() {
         const buildNode = (n: ToolkitNotification, type: NotificationType) => {
+            let icon: Icon & { color?: vscode.ThemeColor }
+            if (type === 'startUp') {
+                icon = getIcon('vscode-question') as Icon
+            } else {
+                icon = getIcon('vscode-alert') as Icon
```

Comment:
could be a ternary, which would allow `cons` and avoids some code.

Reply:
Doing this in ternary form actually causes a runtime error, I suspect because doing the color assignment in the same statement by unwrapping into a new object:
` { ...getIcon(), color: ... }`